src/components/Screen/index.js

```diff
@@ -37,7 +37,7 @@ Split.propTypes = {
   intro: PropTypes.node,
   title: PropTypes.string.isRequired,
   main: PropTypes.node.isRequired,
-  aside: PropTypes.node.isRequired,
+  aside: PropTypes.node,
```

Comment:
Thanks, I should have made this change in https://github.com/eggheadio/egghead-instructor-center/pull/137 and missed it